src/signals/settings/components/Overview/components/__tests__/Overview.test.tsx

```diff
@@ -0,0 +1,145 @@
+import { render } from '@testing-library/react'
+import { withAppContext } from 'test/utils'
+import Overview from '../Overview'
+
+describe('Overview component', () => {
+  it('should render', () => {
+    withAppContext(
+      <Overview
+        showItems={{
+          settings: false,
+          departments: false,
+          groups: false,
+          users: false,
+          categories: false,
+        }}
+      />
+    )
+  })
+
+  it('should show all settings when allowed all', () => {
+    const { queryByTestId } = render(
```

Comment:
Can you use screen.queryByTestId for this and other matchers instead? This is a convention we're using for all new/refactored tests